frontend/SimpleButtons.html

```diff
@@ -21,16 +21,21 @@
     <script src="snap.svg-min.js"></script>
 
     <script src="arm.js"></script>
-    <script src="app.js"></script>
+    <script src="simple.js"></script>
```

Comment:
It looks like simple.js does the same thing as app.js , so maybe we can keep app.js here, and then add a new js file under this in place of oneClickMove that will instantiate the app and arm, and add desired functionality.